spec/ddtrace/contrib/rails/support/rails3.rb

```diff
@@ -13,15 +13,13 @@
 
 # Patch Rails::Application so it doesn't raise an exception
 # when we reinitialize applications.
-Rails::Application.class_eval do
-  class << self
-    def inherited(base)
-      # raise "You cannot have more than one Rails::Application" if Rails.application
-      super
-      Rails.application = base.instance
-      Rails.application.add_lib_to_load_path!
-      ActiveSupport.run_load_hooks(:before_configuration, base.instance)
-    end
+Rails::Application.singleton_class.class_eval do
+  def inherited(base)
+    # raise "You cannot have more than one Rails::Application" if Rails.application
```

Comment:
Unrelated to this change, but this commented out line is a bit suspect 🤔 .

Reply:
![ducks](https://media.giphy.com/media/B37cYPCruqwwg/giphy.gif)